src/App.tsx

```diff
@@ -1,109 +1,75 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { ItemPerPage } from './types/ItemPerPage';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
+const itemsCount: ItemPerPage[] = [3, 5, 10, 20];
```

Comment:
we can make CONSTANTS for it, right? Move it to another file - `utils/constants`